Formula/libdatrie.rb

```diff
@@ -0,0 +1,25 @@
+class Libdatrie < Formula
+  desc "Double-Array Trie Library"
+  homepage "https://linux.thai.net/projects/datrie"
+  url "https://github.com/tlwg/libdatrie/releases/download/v0.2.12/libdatrie-0.2.12.tar.xz"
+  sha256 "452dcc4d3a96c01f80f7c291b42be11863cd1554ff78b93e110becce6e00b149"
+
+  depends_on "autoconf" => :build
+  depends_on "autoconf-archive" => :build
+  depends_on "automake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--enable-shared"
+    system "make", "install-am"
+    system "make", "install-exec"
+  end
+
+  test do
+    system "#{bin}/trietool"
```

Comment:
Any chance of a test that exercises a deeper level of functionality? Thanks!

Reply:
Is it possible to run test cases (make check) inside `test do` as a result of the test functionality?
Thanks.

Reply:
It won't work, I'm afraid. The `testpath` and `buildpath` are separate locations that don't exist beyond the point where `test` or `install` has finished executing.